subsys/wifi/sm.c

```diff
@@ -432,6 +376,8 @@ int sm_sta_init(struct wifimgr_state_machine *sta_sm)
 	sta_sm->state = sta_sm->old_state = WIFIMGR_SM_STA_NODEV;
 	sem_init(&sta_sm->exclsem, 0, 1);
 
+	wifimgr_init_work(&sta_sm->work, wifimgr_sta_event_timeout);
```

Comment:
This routine initializes a workqueue work item, prior to its first use.
Can it be initialized more than one time?